src/cdk-experimental/menu/menu-bar.spec.ts

```diff
@@ -67,6 +70,77 @@ describe('MenuBar', () => {
       expect(spy).toHaveBeenCalledWith(menuItems[0]);
     });
   });
+
+  describe('with submenu', () => {
+    let fixture: ComponentFixture<MenuBarWithMenus>;
+
+    let menus: CdkMenu[];
+    let triggers: CdkMenuItemTrigger[];
+
+    /** opens all menus and asserts they are visible */
+    const openAllMenus = () => {
```

Comment:
Since there's only one menu here, maybe just call this 'openMenu()'. That way it's clearer why you're expecting menus.length to equal 1 in various tests.